doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -296,6 +296,10 @@ Libraries for networking
 
   * Updated the signature of :c:func:`fota_download_start_with_image_type` to accept an integer parameter specifying the PDN ID, which replaces the parameter used to specify the APN.
 
+* :ref:`lib_dfu_target` library:
+  
+  * In preparation of removing socket based mfu from libmodem, changed DFU target modem delta dfu to use the new socketless interface provided by libmodem.
+
```

Comment:
Something like this? Have I put it in the right place? More links/refs I could/should add?

Reply:
LGTM, lets see what @FrancescoSer  says :)

Reply:
Shouldn't also "/libraries/dfu/dfu_target.rst" be updated accordingly? :)